CONTRIBUTING.md

```diff
@@ -598,18 +598,6 @@ There are three types of assets you can build:
 2. `npm run dev-server`: local development assets, with sourcemaps and hot refresh support
 3. `npm run build-instrumented`: instrumented application code for collecting code coverage from Cypress tests
 
-If this type of error comes while building assets(i.e using above commands):
-
-```bash
-Error: You must provide the URL of lib/mappings.wasm by calling SourceMapConsumer.initialize
```

Comment:
Even though i didn't add these docs but I feel `calling SourceMapConsumer.initialize` would still be needed.

Reply:
Upon messing with it, it seems like you shouldn't _have to_ call the initialize method when using React... and that merely installing the `source-map` npm module fixes this. 

I did `unset NODE_OPTIONS` to set my machine back to "stock" and all of the webpack build modes seem to work fine now.